src/components/topic/TopicDirective.js

```diff
@@ -37,6 +37,7 @@ goog.require('ga_topic_service');
               scope.activeTopic = gaTopic.get();
               scope.$applyAsync(function() {
                 element.find('.ga-topic-item').tooltip({
+                  container: 'div.' + modalClsName,
```

Comment:
Maybe we could remove all these toolitps and use the tilte attribute ? what do you think?